torch/nn/functional.py

```diff
@@ -3512,6 +3512,60 @@ def multi_margin_loss(
 """,
 )
 
+_native_pixel_shuffle = _add_docstr(
+    torch._native_pixel_shuffle,
+    r"""
+_native_pixel_shuffle(input, upscale_factor) -> Tensor
+
+Native kernel level implementation of the `pixel_shuffle`.
+This function might become private in future releases, use with caution.
```

Comment:
doc nit: the function is private now since it begins with an underscore

Reply:
This issue has been fixed, remove `_native_pixel_shuffle`.